Code/Tools/AWSNativeSDKInit/source/AWSLogSystemInterface.cpp

```diff
@@ -24,6 +25,9 @@ AZ_POP_DISABLE_WARNING
 
 namespace AWSNativeSDKInit
 {
+    AZ_CVAR(int, bg_awsLogLevel, -1, nullptr, AZ::ConsoleFunctorFlags::Null,
+        "AWSLogLevel used to control verbosity of logging system. Off = 0, Fatal = 1, Error = 2, Warn = 3, Info = 4, Debug = 5, Trace = 6");
```

Comment:
IMHO if we are pesudo defining an enum, we should actually use enums. This may be how other CVARs do this, so will defer to prior art. 

Actual value should be based on Aws::Utils::Logging::LogLevel surely?

Reply:
I don't get your question here, you want to change this cvar type to Aws::Utils::Logging::LogLevel, which is a enum type? if so, I think you still need to pass something like bg_awsLogLevel=3. Then what is the point to change the type here.

Reply:
Nit: In my mind these should relate to the existing enums, using explicit casting to the int values. Because they are decoupled they run the risk of changes breaking them if we ever change or add to the existing log level enums.